src/app/wallet/shared/signature-address-modal/signature-address-modal.component.html

```diff
@@ -50,15 +50,13 @@ <h2 md-dialog-title>Signatures &ndash; Sign/Verify message</h2>
                  [cbContent]="formData.signature" fontSet="ncIcon" fontIcon="nc-copy-2"></md-icon>
       </div><!-- .signature-address-modal-form -->
     </div>
-    <!-- @TODO: icon will be replace with old theme icon  -->
     <md-dialog-actions fxLayoutAlign="end center">
       <button type="reset" md-button (click)="resetForm()">
-        <md-icon>clear</md-icon>
+        <md-icon fontSet="partIcon" fontIcon="part-circle-remove"></md-icon>
```

Comment:
Icon seems a bit squary on the left, is it hidden by something ?
![screenshot_20171107_154704](https://user-images.githubusercontent.com/3384097/32499481-fa845dee-c3d2-11e7-863f-65f5ecdacec4.png)